service-infrastructure/modules.tf

```diff
@@ -284,194 +371,113 @@ module "ecs_warehouse" {
   create_internal_alb           = false
 }
 
-module "rds_warehouse" {
+module "warehouse_database" {
```

Comment:
If we're looking for a like-for-like name with `redis`, maybe using `postgres` instead of `rds` is better given that RDS is an abstraction around several possible RDMSs, and the way that we connect from applications is specific to Postgres not RDS. I do concur with the general point, though.

Reply:
I think postgres/ database makes it more obvious to what it is, so happy with either of them, if we do change this though - we will also need to change the following as well to make them consistent: toggle_database, auth_database and register_api_database